src/librustc_resolve/diagnostics.rs

```diff
@@ -843,6 +843,17 @@ Or:
 let unknown_variable = 12u32;
 let x = unknown_variable; // ok!
 ```
+
+If the item isn't available in the current module, please check you imported it:
+
+```ignore
+use foo::bar;
+bar();
+```
+
+... and check if the item is public, non-private at least, to the current
```

Comment:
Maybe: "If the item you are importing is not defined in some super-module of the current module, then it must also be declared as public (e.g., `pub fn`)."